physx/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 
 #![warn(clippy::all)]
 #![warn(rust_2018_idioms)]
+#![allow(unused_attributes)]
```

Comment:
I'd remove the preceding line instead - it does nothing where it is.

Reply:
The reason I added this was that `#![warn(rust_2018_idioms)]` is present in 32 files. I'll delete it in all of them instead.